src/mappings/helpers.ts

```diff
@@ -141,9 +141,9 @@ export function getAndUpdateUserDailyData(entity: User, timestamp: BigInt): User
 
   dailyData.save()
 
-  return dailyData as UserDailyData
+  return dailyData
 }
 
-export function compoundId(idA: string, idB: string): string {
-  return idA.concat('-').concat(idB)
+export function compoundId(idA: Bytes, idB: Bytes): Bytes {
+  return idA.concat(idB)
```

Comment:
Small caveat with this function, there are some cases (unsure if it happens in this particular subgraph though) where you can end up with ID collision by using this approach (no separator when joining IDs).
It happened in the activity subgraph, where we had an ID generation logic that took an address and 2 numbers (which I can't recall if it was an address + log index and a counter, or the other way around).
You could have multiple collisions as long as `logIndex.concat(counter)` would be equals, which can happen in many different ways, like having a log index of 11 and a count equals to 0, and a log index of 1 and a count equals to 10.

There are 2 different ways of avoiding such collisions, one is to simply use a separator (when using strings it was often times quite easy since you simple used `-`and that's it, but with bytes you can't, so you need to use a random set of characters that you know) or use fixed bytes length for each different type of data (that way, if you know you are having 32 bytes for each number, 11 + 0 and 1 + 10 won't collide, since the padding will be different).